### ISSUE TITLE
[BUG] NET_RAW probe regression

### ISSUE THREAD

**Issue by eddie-knight** [citihub/probr#270]
**Describe the bug**
While modifying the bulk of our probes to drop NET_RAW, we seem to have overdone it and we are dropping NET_RAW from the probes that should be testing whether NET_RAW is rejected.

**To Reproduce**
1. Run Probr for Pod Security Policy only
2. See 3 failures related to NET_RAW

**Expected behavior**
Pods for the NET_RAW scenarios should *not* have NET_RAW dropped.

**Screenshots**
n/a

**Additional context**
n/a

**Comment by lcarrazana** [comment]
A potential solution:
-  Continue to create all pods with default security context, which will drop whatever is in config (so far: NET_RAW)
- For those probes that test specific capabilities, we can override default security context with whatever conditions we are simulating.
  - e.g: If NET_RAW is being tested, and it is dropped by default, we can remove NET_RAW from DROP section during test

**Comment by lcarrazana** [comment]
The three errors mentioned in this bug all belong to the following scenario outline:
```
@k-psp-007
    Scenario Outline: Prevent deployments from running with the NET_RAW capability.
        Given a Kubernetes cluster exists which we can deploy into
        And some system exists to prevent a Kubernetes deployment from running with NET_RAW capability in an existing Kubernetes cluster
        When a Kubernetes deployment is applied to an existing Kubernetes cluster
        And  NET_RAW is marked "<NET_RAW requested>" for the Kubernetes deployment
        Then the operation will "<RESULT>" with an error "<ERRORMESSAGE>"
        But I should not be able to perform a command that requires NET_RAW capability
        And I should be able to perform an allowed command

        Examples:
            | NET_RAW requested | RESULT  | ERRORMESSAGE                                  |
            | True              | Fail    | Containers cant run with NET_RAW capabilities |
            | False             | Succeed |                                               |
            | Not Defined       | Succeed |                                               |
```

This is being addressed in #259

**Comment by lcarrazana** [comment]
After PR #286 regression test showed these tests passing. 
![image](https://user-images.githubusercontent.com/62344673/107791430-efd51680-6d21-11eb-911e-a7991d7ea316.png)
